index.md

```diff
@@ -327,10 +278,10 @@ manage representations independently from each other. This can be desired for
 very large representations, in terms of file size or the amount of files (making
 the root METS difficult to work with).
 
-As a corollary of this division method we define, on the one hand, a
+As a corollary of this division method we define, a
 representation-based division as the separation of representations in different
 folders under the `representations` folder as shown in the example of Figure
-[4](#fig4). And, on the other hand, we define a size-based division as the
+[4](#fig4). And we define a size-based division as the
```

Comment:
And we define a size-based division as the == We also define a size-based division as the

Reply:
done